wpimath/src/main/java/edu/wpi/first/math/system/plant/LinearSystemId.java

```diff
@@ -10,12 +10,67 @@
 import edu.wpi.first.math.numbers.N1;
 import edu.wpi.first.math.numbers.N2;
 import edu.wpi.first.math.system.LinearSystem;
+import edu.wpi.first.units.Angle;
+import edu.wpi.first.units.Dimensionless;
+import edu.wpi.first.units.Distance;
+import edu.wpi.first.units.Mass;
+import edu.wpi.first.units.Measure;
+import edu.wpi.first.units.Mult;
+import edu.wpi.first.units.Per;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.units.Velocity;
+import edu.wpi.first.units.Voltage;
 
 public final class LinearSystemId {
   private LinearSystemId() {
     // Utility class
   }
 
+  /**
+   * Create a state-space model of an elevator system. The states of the system are [position,
+   * velocity]ᵀ, inputs are [voltage], and outputs are [position].
+   *
+   * @param motor The motor (or gearbox) attached to the carriage.
+   * @param mass The mass of the elevator carriage.
+   * @param radius The radius of the elevator's driving drum.
+   * @param G The reduction between motor and drum, as a ratio of output to input.
+   * @return A LinearSystem representing the given characterized constants.
+   * @throws IllegalArgumentException if mass &lt;= 0, radius &lt;= 0, or G &lt;= 0.
+   */
+  public static LinearSystem<N2, N1, N1> createElevatorSystem(
+      DCMotor motor, Measure<Mass> mass, Measure<Distance> radius, Measure<Dimensionless> G) {
```

Comment:
What is `Dimensionless` providing here over a raw `double`? It's mostly useful for converting between multipliers and percentages (eg 100.00% <-> 1.00), and gear ratios are just a scalar multiplier. They're never represented as anything else, so wrapping them in a `Measure` is just extra overhead in my opinion

Reply:
I'll take them out.   

Is there something wrong with my build file that prevents CMake/Build-Linux from completing.  Everything is fine on the local end